plugins/meta/portmap/portmap_suite_test.go

```diff
@@ -40,14 +41,18 @@ func TestPortmap(t *testing.T) {
 	RunSpecs(t, "plugins/meta/portmap")
 }
 
-var echoServerBinaryPath string
+var echoServerBinaryPath, echoClientBinaryPath string
 
 var _ = SynchronizedBeforeSuite(func() []byte {
-	binaryPath, err := gexec.Build("github.com/containernetworking/plugins/pkg/testutils/echosvr")
+	serverBinaryPath, err := gexec.Build("github.com/containernetworking/plugins/pkg/testutils/echo/server")
 	Expect(err).NotTo(HaveOccurred())
-	return []byte(binaryPath)
+	clientBinaryPath, err := gexec.Build("github.com/containernetworking/plugins/pkg/testutils/echo/client")
+	Expect(err).NotTo(HaveOccurred())
+	return []byte(fmt.Sprintf("%s,%s", serverBinaryPath, clientBinaryPath))
```

Comment:
`strings.Join` ?

Reply:
Done, don't have a strong opinion on that since the list is two elements.